code/modules/urist/items/uristmed.dm

```diff
@@ -39,3 +39,174 @@ Space for all Urist-done, non-pill medical items. Please keep it tidy, as usual.
+							else return 0
+
+
+					else
+						user <<	"<span class='warning'> This wound is not bleeding, use more advanced medical gear to heal the [W.desc] on [M]'s [affecting.display_name].</span>"
+						return 0
+		else
+			if (can_operate(H))        //Checks if mob is lying down on table for surgery
+				if (do_surgery(H,user,src))
+					return
+			else
+				user << "<span class='notice'>The [affecting.display_name] is cut open, you'll need more than a [src]!</span>"
+				return 0
+
+
+/obj/item/clothing/suit/verb/rip(mob/user as mob)
+	set name = "Rip Apart Suit"
+	set category = "Object"
+	set src in usr
+
+	var/mob/living/carbon/human/H = src.loc
+
+	if(!usr.canmove || usr.stat || usr.restrained())
+		return 0
+	if(!istype(H))
+		return 0
+	if(H.wear_suit == src)
+		user << "<span class='warning'>Deciding not to channel your inner barbarian, you think it's a good idea to take off your [src] first.</span>"
+		return
+
+	if(istype(src, /obj/item/clothing/suit/space) || istype(src, /obj/item/clothing/suit/armor) || istype(src, /obj/item/clothing/suit/wizrobe)) //add other protected instances as needed
+		return 0
+	else
+		user.visible_message( 	"<span class='notice'> [user] starts ripping [src] apart into rags!</span>", \
+											"<span class='notice'> You start ripping [src] apart into rags!</span>" )
+		spawn(100)
+			user << "<span class='notice'>You have ripped [src] into pieces, yielding two rags.</span>"
+
+			playsound(src.loc, 'sound/weapons/cablecuff.ogg', 100, 1)
+			spawn(5)
+				playsound(src.loc, 'sound/weapons/cablecuff.ogg', 100, 1)
+
+			new /obj/item/weapon/clothesrag(get_turf(src))
+			new /obj/item/weapon/clothesrag(get_turf(src))
+
+			del(src)
+
+			H.update_inv_l_hand()
+			H.update_inv_r_hand()
+
+
+/obj/item/clothing/under/verb/rip(mob/user as mob)
+	set name = "Rip Apart Jumpsuit"
+	set category = "Object"
+	set src in usr
+	var/mob/living/carbon/human/H = src.loc
+
+	if(!usr.canmove || usr.stat || usr.restrained())
+		return 0
+/*	if(istype(src, /obj/item/clothing/under/BADSTUFFGOESHERE //I couldn't think of any item types here that shouldn't work, but leaving it here in case it's needed later
+		return 0*/
+	if(!istype(H))
+		return 0
+	if(H.w_uniform == src)
+		user << "<span class='warning'>Deciding not to channel your inner barbarian, you think it's a good idea to take off your [src] first.</span>"
+		return
+
+	else
+		user.visible_message( 	"<span class='notice'> [user] starts ripping [src] apart into a rag!</span>", \
+											"<span class='notice'> You start ripping [src] apart into a rag!</span>" )
+		spawn(100)
+			user << "<span class='notice'>You have ripped [src] into pieces, yielding a rag.</span>"
+
+			playsound(src.loc, 'sound/weapons/cablecuff.ogg', 100, 1)
+			spawn(5)
+				playsound(src.loc, 'sound/weapons/cablecuff.ogg', 100, 1)
+
+			new /obj/item/weapon/clothesrag(get_turf(src))
+
+			del(src)
```

Comment:
Needs to check hastie to see if there's an accessory to leave behind first, so medals are not ripped in pieces, and more importantly, holsters with guns in them.